### PR TITLE
Fixes #531 version 4

### LINKED ISSUE #531: Target version 4 of R

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -17,7 +17,7 @@ BugReports:
     https://github.com/open-systems-pharmacology/ospsuite-r/issues
 Depends: 
     R (>= 3.5),
-    rClr (>= 0.8.3)
+    rClr (>= 0.9.1)
```
Comment (msevestre) [inline DESCRIPTION:20]: @PavelBal @Yuri05  Is this what we need to do?
Target 0.9.1 but say that we depend on R 3.5?
Appveyor will build with v4.0.x

Comment (msevestre) [reply DESCRIPTION:20]: let's undo this

File: DESCRIPTION [hunk 2]
```diff
@@ -9,15 +9,15 @@ Authors@R:
...
+Description: The ospsuite-R package provides the functionality of loading, manipulating, and simulating the simulations 
+  created in the Open Systems Pharmacology Software tools PK-Sim and MoBi.
 License: GPL-2 | file LICENSE
 URL: https://github.com/open-systems-pharmacology/ospsuite-r
 BugReports: 
     https://github.com/open-systems-pharmacology/ospsuite-r/issues
 Depends: 
     R (>= 3.5),
-    rClr (>= 0.9.1)
+    rClr (>= 0.8.3)
```
Comment (PavelBal) [inline DESCRIPTION]: Let's use 0.9.1 as we now have releases of the version for R3 and R4, and 0.9.1 has a bug fix that 0.8.3 does not.

Comment (msevestre) [reply DESCRIPTION]: YEs of course. Good idea

**Pull request merged**